src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,30 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, selectedTab, onTabSelected }) => (
+  <div data-cy="TabsComponent">
+    <div className="tabs is-boxed">
+      <ul>
+        {tabs.map(tab => (
+          <li
+            key={tab.id}
+            className={tab === selectedTab ? 'is-active' : ''}
```

Comment:
Use classname library instead, also use a variable for duplicate checks (tab === selectedTab and on line 15)